database/insertData.js

```diff
@@ -6,19 +6,18 @@ const graphQLendpoint = config.graphQLendpoint
 
 const queries = [
   // Template A -- Test - General Registration (Feature showcase)
-  `mutation {
+  `
+
```

Comment:
Any particular reason for this extra whitespace?

Reply:
Oops! Removing this